packages/api/cms-api/src/auth/auth.resolver.ts

```diff
@@ -22,17 +22,20 @@ export function createAuthResolver(CurrentUser: Type<CurrentUserInterface>): Typ
 
         @Mutation(() => String)
         async currentUserSignOut(@Context("req") req: IncomingMessage): Promise<string | null> {
-            let redirectUri = this.moduleConfig.postLogoutRedirectUri || "/";
+            let signOutUrl = this.moduleConfig.postLogoutRedirectUri || "/";
 
             if (req.headers["authorization"] && this.config.endSessionEndpoint) {
                 const url = new URL(this.config.endSessionEndpoint);
                 url.search = new URLSearchParams({
                     id_token_hint: req.headers["authorization"].substring(7),
-                    post_logout_redirect_uri: redirectUri,
+                    post_logout_redirect_uri: signOutUrl,
                 }).toString();
-                redirectUri = url.toString();
+                signOutUrl = url.toString();
             }
-            return `/oauth2/sign_out?rd=${encodeURIComponent(redirectUri)}`;
+            if (this.moduleConfig.formatSignOutUrl) {
+                signOutUrl = this.moduleConfig.formatSignOutUrl(signOutUrl);
```

Comment:
Isn't a concrete implementation missing in Demo that does the `/oauth2/sign_out?rd=${encodeURIComponent(redirectUri)}` part?

Reply:
Since the demo doesn't use oauth2-proxy the signOutUrl don't has to be set. But there will be a default implementation in the starter.